buildenv/jenkins/variables/defaults.yml

```diff
@@ -207,7 +207,10 @@ ppc64le_linux:
   extra_configure_options:
     all: '--enable-jitserver'
   build_env:
-    cmd: 'source /home/jenkins/set_gcc7.5.0_env'
+    cmd:
```

Comment:
To avoid having to make future changes we should reverse the logic. Explicitly set 8, 11, 17, and use `all` for 18 and future builds.

Reply:
done